Loaded 1000 rollouts
Loaded 1000 rollouts
> /home/aurora/handful-of-trials-pytorch/dynamics_model/new_model/data/dataset.py(55)__init__()
-> sample = self.base[traj_idx]
> /home/aurora/handful-of-trials-pytorch/dynamics_model/new_model/data/dataset.py(57)__init__()
-> state_seq = sample[2]
> /home/aurora/handful-of-trials-pytorch/dynamics_model/new_model/data/dataset.py(59)__init__()
-> seq_len = state_seq.shape[0]
torch.Size([4, 8])
1
*** AttributeError: '_RolloutDataset' object has no attribute 'num_history'
*** AttributeError: '_RolloutDataset' object has no attribute 'num_historyq'
Traceback (most recent call last):
  File "/home/aurora/handful-of-trials-pytorch/dynamics_model/new_model/train.py", line 144, in <module>
    main(sys.argv[1])
  File "/home/aurora/handful-of-trials-pytorch/dynamics_model/new_model/train.py", line 63, in main
    train_ds, val_ds = load_pusht_dataset(cfg.data)
  File "/home/aurora/handful-of-trials-pytorch/dynamics_model/new_model/data/dataset.py", line 40, in load_pusht_dataset
    train_ds = _RolloutDataset(train_base, data_cfg.num_hist)
  File "/home/aurora/handful-of-trials-pytorch/dynamics_model/new_model/data/dataset.py", line 59, in __init__
    seq_len = state_seq.shape[0]
  File "/home/aurora/handful-of-trials-pytorch/dynamics_model/new_model/data/dataset.py", line 59, in __init__
    seq_len = state_seq.shape[0]
  File "/home/aurora/anaconda3/envs/pusht/lib/python3.10/bdb.py", line 90, in trace_dispatch
    return self.dispatch_line(frame)
  File "/home/aurora/anaconda3/envs/pusht/lib/python3.10/bdb.py", line 115, in dispatch_line
    if self.quitting: raise BdbQuit
bdb.BdbQuit